rest/replicator_test.go

```diff
@@ -3933,6 +3932,23 @@ func TestSGR2TombstoneConflictHandling(t *testing.T) {
 	}
 
 	for _, test := range tombstoneTests {
+		// requireTombstone validates tombstoned revision.
+		requireTombstone := func(bucket *base.TestBucket, docID string) {
+			var rawBody db.Body
+			_, err := bucket.Get(docID, &rawBody)
+			if base.TestUseXattrs() {
+				require.Error(t, err)
```

Comment:
`t` being used inside this function would reference the parent test, not each subtest, which can make it difficult to find where exactly the test failed.

Test util functions generally take a `testing.T/testing.TB` as the first parameter to cover this: E.g:

![Screenshot 2021-01-19 at 15 48 19](https://user-images.githubusercontent.com/1525809/105058081-c51ed780-5a6d-11eb-8615-bfc6b2c2dc00.png)

Reply:
Make sense, this is added in the latest commit.